statsmodels/survey/examples/survey_df.csv

```diff
@@ -0,0 +1,184 @@
+"dnum","pw","api00","api99"
```

Comment:
184 lines is not large, so it's not a problem for the repo.
We don't want to have huge data files, or autogenerated content like the output in notebook files in the repo. But some test files are necessary and if we want prediction or residuals, then it will add to the csv content in the repo.

(The other thing I find misleading is if there are large data objects/arrays and line counts in python modules for unit tests, because it distorts the overall line count for the package. But again some test files like the tests/results files are necessary and they won't change much once they have been added, as long as they still stay within a reasonably small fraction of overall line count.)